sample-functions/NodeBaseFunction/Dockerfile.armhf

```diff
@@ -0,0 +1,20 @@
+FROM arm32v6/alpine:3.6
+
+RUN apk --update add nodejs nodejs-npm
```

Comment:
This is my bad for using --update elsewhere. Please can we change to --no-cache? It means less left over in the final image.